### DISCUSSION TITLE
Product carousel need more filters

### DISCUSSION THREAD

**Discussion by KBM90** [bagisto/bagisto#9954] [Ideas]
We need more filters or a field to choose which filter to apply for products carousel in themes , beacause the filters you're providing are not very helpful,for example in my store i have many categories so i want a some products carousel that shows only a category's products . 
And it will better if we can make filters based on attributes

**Comment by amit-webkul** [comment]
@KBM90 

We will review this matter with our internal team to determine the next steps. We will keep you informed of any updates accordingly.

**Comment by devansh-webkul** [comment]
You can simply create an attribute and make it filterable by enabling the "Show in Layered Navigation" option. After that, the attribute will also become visible in the product carousel.
